source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -105,7 +105,15 @@ After the record is created wait for up to an hour for the CloudFront distributi
 
 #### CDN Configuration Options
 
```

Comment:
I wonder whether this section needs an intro to explain what the contents in the table is/relates to...